packages/react-components/react-breadcrumb/src/utils/partitionBreadcrumbItems.ts

```diff
@@ -42,6 +42,8 @@ export const partitionBreadcrumbItems = <T>(
     startDisplayedItems = startDisplayedItems.slice(0, overflowIndex);
     overflowItems = items.slice(overflowIndex, menuLastItemIdx);
     endDisplayedItems = items.slice(menuLastItemIdx, itemsCount);
+  } else {
+    endDisplayedItems = items.slice(overflowIndex, itemsCount);
   }
```

Comment:
Hello @GordonSmith , thank you for submitting this PR. Could you please, also, add this case to the unit tests in `partitionBreadcrumbItems.test.ts`?